src/containers/sprite-info.jsx

```diff
@@ -25,7 +25,10 @@ class SpriteInfo extends React.Component {
         };
     }
     handleKeyPress (e) {
-        if (e.key === 'Enter') this.handleFlush();
+        if (e.key === 'Enter') {
+            this.handleFlush();
+            e.target.blur();
+        }
```

Comment:
That came from a design review of the feature. We thought it would be more expected for the field to lose focus when you hit enter, otherwise it is not as obvious that hitting enter did anything.

Now that you mention it, I think the blur is the only thing that we need to do, since the flushing will happen on blur anyway...